lua/lazyvim/plugins/extras/lang/angular.lua

```diff
@@ -14,6 +14,18 @@ return {
       if type(opts.ensure_installed) == "table" then
         vim.list_extend(opts.ensure_installed, { "angular", "scss" })
       end
+      vim.filetype.add({
+        pattern = {
+          [".*%.component%.html"] = "angular.html",
+          [".*%.container%.html"] = "angular.html",
+        },
+      })
+      vim.api.nvim_create_autocmd("FileType", {
+        pattern = "angular.html",
+        callback = function()
+          vim.treesitter.language.register("angular", "angular.html")
+        end,
```

Comment:
IS this really needed to do in an autocmd?
Why not just register it?

Reply:
Indeed, you are right, this is not necessary.
Removed the wrapping of registration with autocmd.